test/fixtures/websockets.html

```diff
@@ -0,0 +1,14 @@
+<div id='result'></div>
+<div id='errorTime'></div>
+<script>
+  // This should get blocked by adblock if it is on
+  var exampleSocket = new WebSocket("ws://ag.innovid.com/dv/sync?tid=2");
```

Comment:
not sure i understand. the webpack ws URL is not in adblock, so it shouldn't get blocked?